crates/polars-core/Cargo.toml

```diff
@@ -63,6 +63,7 @@ rolling_window = []
 rank = []
 diff = []
 pct_change = ["diff"]
+search_sorted = []
```

Comment:
Search sorted should not be moved into `polars-core`. Rather `is_in` should be moved into `polars-ops`.